src/lib.rs

```diff
@@ -16,8 +17,8 @@ pub fn set_locale(locale: &str) {
 }
 
 /// Get current locale
-pub fn locale() -> Arc<String> {
-    CURRENT_LOCALE.clone_string()
+pub fn locale() -> impl Deref<Target = str> {
+    CURRENT_LOCALE.as_str()
```

Comment:
Do not change locale method for keep back forward compatibility.

Reply:
`triomphe::Arc` is not the same as `std::sync::Arc`.

Reply:
@varphone What do you think about this?

Last week, we just release v3 with breaking changes, I don't want to do it again.

Would we have any way to keep backward compatible?

Reply:
It would always be possible to omit the second commit. It is only meant to negate the runtime impact of the first commit.